pricepoint_intel/geospatial/risk_framework.py

```
@@ -0,0 +1,1076 @@
+            entity_type="region",
+            risk_level=risk_level,
+            risk_score=min(100, risk_score),
+            factors=factors,
+            recommendations=recommendations,
+        )
+
+    def find_optimal_vendors(
+        self,
+        latitude: float,
+        longitude: float,
+        sku_ids: Optional[list[str]] = None,
```

Ruff (0.14.6)

Unused method argument: `sku_ids`

(ARG002)